drivers/SmartThings/zigbee-switch/src/aqara/init.lua

```diff
@@ -0,0 +1,168 @@
+local capabilities = require "st.capabilities"
+local clusters = require "st.zigbee.zcl.clusters"
+local cluster_base = require "st.zigbee.cluster_base"
+local data_types = require "st.zigbee.data_types"
+local SinglePrecisionFloat = require "st.zigbee.data_types".SinglePrecisionFloat
+
+local maxPowerPreferenceId = "stse.maxPower"
+local restorePowerStatePreferenceId = "stse.restorePowerState"
+
+local Groups = clusters.Groups
+local SimpleMetering = clusters.SimpleMetering
+local ElectricalMeasurement = clusters.ElectricalMeasurement
+
+local MFG_CODE = 0x115F
+local PRIVATE_CLUSTER_ID = 0xFCC0
+local PRIVATE_ATTRIBUTE_ID = 0x0009
+
+local PREF_CLUSTER_ID = 0xFCC0
+local PREF_ATTRIBUTE_MAX_POWER_ID = 0x020B
+local PREF_ATTRIBUTE_RESTORE_POWER_STATE_ID = 0x0201
+
+local PREF_VALUE_MAX_POWER_DEFAULT = 23
+local PREF_VALUE_RESTORE_POWER_STATE_DEFAULT = false
+
+local LAST_REPORT_TIME = "LAST_REPORT_TIME"
+
+local FINGERPRINTS = {
+  { mfr = "LUMI", model = "lumi.plug.maeu01" }
+}
+
+local max_power_table = {
```

Comment:
I assume these values are Watts, but maybe there could be a comment or name change to confirm Also, could you comment on each line listing what that actual number is so a human can understand at first glance?

Reply:
Sure. I added a description.